hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/RestoreDriver.java

```diff
@@ -124,8 +124,8 @@ private int parseAndRun() throws IOException {
 
     if (cmd.hasOption(OPTION_YARN_QUEUE_NAME)) {
       String queueName = cmd.getOptionValue(OPTION_YARN_QUEUE_NAME);
-      // Set system property value for MR job
-      System.setProperty("mapreduce.job.queuename", queueName);
+      // Set MR job queuename to configuration
+      getConf().set("mapreduce.job.queuename", queueName);
```

Comment:
Hi @2005hithlj  I was curious with one doubt? Does not simply passing -Dmapreduce.job.queuename=YOUR_QUEUE_NAME as args to the MapReduce job work?

Reply:
@NihalJain  Thanks for your review. Currently, the backup command uses the -q option to specify the yarn queue, but it does not take effect, this PR is intended to fix this problem. Additionally, the -D way is not intuitive for users, and we do not have any doc to explain how to use this approach effectively.